src/ads/ad.js

```diff
@@ -11,6 +11,9 @@ class Ad {
   _posterUrl: ?string;
   _skipOffset: ?number;
   _linear: ?boolean;
+  _width: number;
```

Comment:
Set default values since this is not supported on cast ads.

Reply:
done